### DISCUSSION TITLE
Development Containers (for installing native dependencies such as PostgreSQL)

### DISCUSSION THREAD

**Discussion by brillout** [vikejs/bati#282] [Integration request]
### Name

Development Containers

### Homepage

https://containers.dev

**Comment by brillout** [comment]
It can be used with many tools [such as devbox](https://containers.dev/supporting#jetify-devbox).

This means that Bati can then provide a deterministic dev environment.

See also:
 - https://news.ycombinator.com/item?id=39696038
 - https://news.ycombinator.com/item?id=39692801

**Reply by magne4000** [reply]
> This means that Bati can then provide a deterministic dev environment.

It will not be deterministic, even if specifying a version when: `pnpm create bati@0.0.333`, because of sub dependencies ranges. But it shouldn't really matter
